### PR TITLE
Add new assessment pages [RAM-258, RAM-260]

### PR DESCRIPTION
**Pull request by DigitalSuey**

### CODE REVIEW

File: _pages/cybersecurity.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+layout: blog
+permalink: /cybersecurity/
+title: Cyber Security
+description: 
+keywords: cybersecurity
+---
+
+<h1 class="mb-4">Cyber Security</h1>
+<p class="subheading-small">
+  Lorem ipsum dolor sit amet consectetur adipisicing elit.
```
Comment (jsntv200) [inline _pages/cybersecurity.html:11]: Assume you're aware of the Lorem?

**Review by jsntv200 — APPROVED** [review]
Looks good to me.

File: _pages/cybersecurity.html (hunk 1 above)
Comment (DigitalSuey) [reply _pages/cybersecurity.html]: Yep, updated in [8da329d](https://github.com/red-ant/redant-home/pull/316/commits/8da329db23ca185605dc286adb31380766741657)

**Comment by Anshar11** [comment]
>

**Comment by Anshar11** [comment]
> 

80ad1478872b5e290eb7b0f6e3c7faac40c9d168

**Comment by Anshar11** [comment]
> 

80ad1478872b5e290eb7b0f6e3c7faac40c9d168

**Comment by Anshar11** [comment]
> 

20fc117d9ec610510d7481a6d264b3ac886e1fe2

**Comment by Anshar11** [comment]
> 

1f45b91f54bc118b44a0d2dd001bd80354f2f13d

**Comment by Anshar11** [comment]
> Looks good to me.

File: _book_call/cybersecurity.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: book-call
+permalink: /cybersecurity/book-call/
```
Comment (benstill) [inline _book_call/cybersecurity.md:3]: can you change to cyber-security

Comment (DigitalSuey) [reply _book_call/cybersecurity.md]: Fixed in https://github.com/red-ant/redant-home/pull/316/commits/b63605999833633bf74ec85b0203876b4ca2a801

**Pull request merged**